k8s/bigchaindb/bigchaindb-dep.yaml

```diff
@@ -59,8 +41,15 @@ spec:
           value: 3CjmRhu718gT1Wkba3LfdqX5pfYuBdaMPLd7ENUga5dm
         - name: BIGCHAINDB_BACKLOG_REASSIGN_DELAY
           value: "120"
-        - name: BIGCHAINDB_KEYRING
-          value: ""
+        - name: BIGCHAINDB_DATABASE_MAXTRIES
+          value: "3"
+        - name: BIGCHAINDB_DATABASE_CONNECTION_TIMEOUT
+          value: "120"
+        - name: BIGCHAINDB_LOG_LEVEL_CONSOLE
+          value: debug
+        # The following env var is not required for the bootstrap/first node
+        #- name: BIGCHAINDB_KEYRING
+        #  value: ""
```

Comment:
There are several BigchainDB Server configuration settings that can be left at their default values besides the BIGCHAINDB_KEYRING. I figured you were including _all_ of the settings just to be explicit, but I suppose the list of settings has gotten rather long lately... I would be okay with _only_ including the ones which should be changed from their default values. The configuration settings and their defaults are all documented on the page: https://docs.bigchaindb.com/projects/server/en/v0.10.1/server-reference/configuration.html